blenderproc/python/types/LightUtility.py

```diff
@@ -243,3 +243,28 @@ def get_type(self, frame: Optional[int] = None) -> str:
         """
         with KeyFrame(frame):
             return self.blender_obj.data.type
+
+
+def get_all_blender_light_objects() -> List[bpy.types.Object]:
+    """ Retrieves all light objects in the current Blender scene.
+
+    :return: A list of all light objects in the current scene.
+    """
+    return [obj for obj in bpy.context.scene.objects if obj.type == 'LIGHT']
+
+
+def convert_to_lights(blender_objects: List[bpy.types.Object]) -> List[Light]:
```

Comment:
Should we maybe add the latter two functions to `blenderproc/api/lighting/__init__.py` to make them available via the api?